### PR TITLE
Clarified some things in the intro page for the MNIST notebook

### PR DESCRIPTION
**Pull request by sarahmaddox**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1731)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview:
https://deploy-preview-1731--competent-brattain-de2d6d.netlify.com/docs/gke/gcp-e2e/

/assign @jlewi

File: content/docs/gke/gcp-e2e.md [hunk 1]
```diff
@@ -72,7 +73,8 @@ In summary:
 * Setting up [Kubeflow][kubeflow] in a [GKE][kubernetes-engine]
   cluster.
 
-* Testing the code locally using a [Jupyter notebook][jupyter-notebook].
+* Testing the code locally using a
```
Comment (jlewi) [inline content/docs/gke/gcp-e2e.md]: Should we delete this? I don't think running the code locally anymore applies. The notebook is designed to run in the cluster not their local machine. They would probably have problems running it locally

**Comment by jlewi** [comment]
Thanks @sarahmaddox one minor suggestion.

File: content/docs/gke/gcp-e2e.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/gke/gcp-e2e.md]: Good idea. Done.

**Comment by sarahmaddox** [comment]
Thanks @jlewi I've made the change based on your suggestion. PTAL.

**Pull request merged**